exdb/views.py

```diff
@@ -1,12 +1,51 @@
 from django.views.generic import TemplateView, ListView
+from django.views.generic.edit import CreateView
+from django.core.urlresolvers import reverse
+from django.utils import timezone
+
 from exdb.models import Experience
+from .forms import ExperienceSubmitForm, ExperienceSaveForm
+
 
 class WelcomeView(TemplateView):
     template_name = 'exdb/welcome.html'
 
+
+class CreateExperienceView(CreateView):
+    model = Experience
+    template_name = 'exdb/create-experience.html'
+
+    def get_success_url(self):
+        return reverse('welcome')
+
+    def get_form_kwargs(self):
+        return super(CreateExperienceView, self).get_form_kwargs()
```

Comment:
If all get_form_kwargs does is call the super, it doesn't need to be re-defined.